DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -252,8 +252,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="16">
-   <version ClassVersion="17" checksum="1633877368"/>
+  <class name="pat::PackedCandidate" ClassVersion="17">
+    <version ClassVersion="17" checksum="1257500115"/>
```

Comment:
mhh then you did already update in the last rebase the checksum including
the changed that lead to current "17" ?
if so just increase the ClassVersion to 18 keeping 1257500115 as checksum

On Sun, Mar 15, 2015 at 7:39 PM, Jason Lee <notifications@github.com> wrote:

> In DataFormats/PatCandidates/src/classes_def_objects.xml
> <https://github.com/cms-sw/cmssw/pull/8122#discussion_r26450577>:
>
> > @@ -252,8 +252,8 @@
> >     <version ClassVersion="11" checksum="1239840459"/>
> >    </class>
> >
> > -  <class name="pat::PackedCandidate" ClassVersion="16">
> > -   <version ClassVersion="17" checksum="1633877368"/>
> > +  <class name="pat::PackedCandidate" ClassVersion="17">
> > +    <version ClassVersion="17" checksum="1257500115"/>
>
> Hi David, I've tried rebuilding and "scram b updateclassversion" and
> didn't see any changes. How can I get the checksum for 18?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8122/files#r26450577>.
>